tutorials/arvo/ames.md

```diff
@@ -111,3 +111,42 @@ ducts as unique keys.  Stop tracking a peer by sending Ames a
 `%jilt` `$task`.
 
 Debug output can be adjusted using `%sift` and `%spew` `$task`'s.
+
+## Packet structure
+
+Ames datagram packets are handled as nouns internally by Arvo but as serial data
+by Unix. Here we document their serial structure.
+For further information, see `+encode-packet` and `+decode-packet` in `ames.hoon`.
+
+There is a 32-bit header followed by a variable width body.
+
+### Header
+
+The 32-bit header is given by the following data:
+
+ - 3 bits: Ames `protocol-version`,
+ - 20 bits: a checksum as a truncated insecure hash of the body, done with
+   `+mug`,
+ - 2 bits: the bit width of the sender address encoded as a 2-bit enum,
+ - 2 bits: the bit width of the receiver address encoded as a 2-bit enum,
+ - 1 bit: whether the packet is encrypted or not,
+ - 4 bits: unused.
+ 
+ Every packet between Azimuth addresses are encrypted. The only unencrypted
```

Comment:
> Azimuth addresses

Aren't we saying "Urbit IDs" now? Especially in these kinds of contexts.

(Also you say "address" in this section, but "`@p`" below.

Reply:
I changed it to @p https://github.com/urbit/docs/pull/943/commits/48c4209a12e4f4c57bc97871e7e028b919c56fec. I'm curious though, are comets considered to be Urbit IDs since they aren't on-chain?

Reply:
Actually, the pedant in me is bothered since @p is a data type, not the ship itself, so now I changed it to ships :P